pkg/transformer/passthrough.go

```diff
@@ -8,19 +8,7 @@ type passThrough struct{}
 
 // Transform a kafka Message
 func (p passThrough) Transform(src *kafka.Message) []*kafka.Message {
-	topic := *src.TopicPartition.Topic + "-passthrough"
-
-	msg := &kafka.Message{
-		TopicPartition: kafka.TopicPartition{
-			Topic:     &topic,
-			Partition: kafka.PartitionAny,
-		},
-		Value:   src.Value,
-		Key:     src.Key,
-		Headers: src.Headers,
-	}
-
-	return []*kafka.Message{msg}
+	return []*kafka.Message{src}
```

Comment:
i think we should get ride of this "default" transformer and connect the consumer out channel with the producer/projector in channel directly